server/src/main/java/org/elasticsearch/index/engine/CommitStats.java

```diff
@@ -69,6 +69,13 @@ public String getId() {
         return id;
     }
 
+    /**
+     * A raw version of the commit id (see {@link SegmentInfos#getId()}
+     */
+    public Engine.CommitId rawCommitId() {
```

Comment:
can we call this `getRawCommitId()` instead?

Reply:
I pushed https://github.com/elastic/elasticsearch/pull/28464/commits/8974f60e50dfabf4c1f93e70eabdad86ab458e77